src/transformers/modeling_tf_longformer.py

```diff
@@ -170,24 +168,20 @@ def call(
         # normalize query
         query_vectors /= tf.math.sqrt(tf.constant(self.head_dim, dtype=tf.dtypes.float32))
 
-        query_vectors = tf.transpose(
```

Comment:
get rid of `tf.transpose` since `tf.transpose` allocates new tenser -> is actually quite costly the function